xsd/PublicationDelivery.genkeyref.xsd

```diff
@@ -1824,7 +1844,11 @@
       <xsd:field xpath="@id"/>
       <xsd:field xpath="@version"/>
     </xsd:key>
-    <!-- Generation of deep key suppressed, key: Equipment_Key -->
+    <xsd:key name="Equipment_Key_Deep">
+      <xsd:selector xpath=".//netex:AccessEquipment | .//netex:AccessVehicleEquipment | .//netex:ActivatedEquipment | .//netex:ActualVehicleEquipment | .//netex:AssistanceBookingService | .//netex:AssistanceService | .//netex:BatteryEquipment | .//netex:BedEquipment | .//netex:CarPoolingService | .//netex:CateringService | .//netex:ChauffeuredVehicleService | .//netex:CommonVehicleService | .//netex:CommonVehicleService_ | .//netex:CommunicationService | .//netex:ComplaintsService | .//netex:CrossingEquipment | .//netex:CustomerService | .//netex:CycleStorageEquipment | .//netex:EntranceEquipment | .//netex:EntranceSensor | .//netex:Equipment | .//netex:EscalatorEquipment | .//netex:GeneralSign | .//netex:HeadingSign | .//netex:HelpPointEquipment | .//netex:HireService | .//netex:InstalledEquipment | .//netex:LeftLuggageService | .//netex:LiftCallEquipment | .//netex:LiftEquipment | .//netex:LocalService | .//netex:LostPropertyService | .//netex:LuggageLockerEquipment | .//netex:LuggageService | .//netex:LuggageSpotEquipment | .//netex:MeetingPointService | .//netex:MobilityService | .//netex:MobilityService_ | .//netex:MoneyService | .//netex:OnlineService | .//netex:OtherPlaceEquipment | .//netex:PassengerBeaconEquipment | .//netex:PassengerEquipment | .//netex:PassengerInformationEquipment | .//netex:PassengerSafetyEquipment | .//netex:PlaceEquipment | .//netex:PlaceLighting | .//netex:PlaceSign | .//netex:QueueingEquipment | .//netex:RampEquipment | .//netex:RefuellingEquipment | .//netex:RetailDevice | .//netex:RetailService | .//netex:RoughSurface | .//netex:RubbishDisposalEquipment | .//netex:SanitaryEquipment | .//netex:SeatEquipment | .//netex:SeatingEquipment | .//netex:SensorEquipment | .//netex:SensorEquipment_ | .//netex:ShelterEquipment | .//netex:SignEquipment | .//netex:SiteEquipment | .//netex:SpotEquipment | .//netex:SpotEquipment_ | .//netex:SpotSensor | .//netex:StairEquipment | .//netex:StaircaseEquipment | .//netex:TaxiService | .//netex:TicketValidatorEquipment | .//netex:TicketingEquipment | .//netex:TicketingService | .//netex:TravelatorEquipment | .//netex:TrolleyStandEquipment | .//netex:VehicleChargingEquipment | .//netex:VehiclePoolingService | .//netex:VehiclePoolingService_ | .//netex:VehicleReleaseEquipment | .//netex:VehicleRentalService | .//netex:VehicleSharingService | .//netex:WaitingEquipment | .//netex:WaitingRoomEquipment | .//netex:WheelchairVehicleEquipment"/>
```

Comment:
Notice the `.//netex:VehiclePoolingService_` is this connect, or should abstracted types be omitted?

But it also unconvers that we need redo it for the _Abstract version.